TapOrHoldCounter/MainVC.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  MainVC.swift
+//  TapOrHoldCounter
+//
+//  Created by vmalikov on 1/28/16.
+//  Copyright © 2016 justForFun. All rights reserved.
+//
+
+import UIKit
+
+class MainVC: UIViewController {
```

Comment:
I prefer to use full names as they are more descriptive. Same as Apple always uses full names like UIViewController, UITableViewCell, etc.